src/status_im/ui/screens/chat/components/input.cljs

```diff
@@ -144,13 +144,9 @@
 
 (defn clear-input [chat-id refs]
   (hide-send refs)
-  (if (get @mentions-enabled chat-id)
-    (do
-      (swap! mentions-enabled dissoc chat-id)
-      ;;we need this timeout, because if we clear text input and first index was a mention object with blue color,
-      ;;after clearing text will be typed with this blue color, so we render white text first and then clear it
-      (js/setTimeout #(reset-input refs chat-id) 50))
```

Comment:
hi @flexsurfer, Do we still need this timeout? I am not able to reproduce this issue anymore.

Reply:
is it reason of the bug? i mean usually its better to keep such changes because who knows

Reply:
interestingly, this is not the cause, but still is fix :D

`hide-send` is disabling send button, but as we are calling `reset-input` delayed, rerender again enabling send button.
fixed enabling of `send-button`. Thank you